src/components/IconButton/src/index.tsx

```diff
@@ -35,6 +35,11 @@ export interface IconButtonProps extends BaseProps {
    * `small` is equivalent to the dense button styling.
    */
   size?: 'small' | 'medium';
+
+  /**
+   * The tab-index of the IconButton.
+   */
+  tabIndex?: number;
```

Comment:
I'm fine with exposing this property. Users should be able to decide for themselves how to use a component under their own circumstances.

Reply:
Move this to baseprops